examples/1.0.0/bnpl-openapi.yaml

```diff
@@ -0,0 +1,457 @@
+                        allOf:
+                          - $ref: '#/components/schemas/CustomerUri'
+                    additionalProperties: false
+                additionalProperties: false
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+  /loan-transactions:
+    post:
+      summary: Initiate a new loan transaction
+      description: |
+        Initiate a new loan based on customer details and in-scope products.
+
+        For the sake of this example:
+
+        * There is one error response, defined using the `default` keyword.
+      operationId: createBnplTransaction
+      requestBody:
+        content:
+          application/json:
+            schema:
+              description: Properties for the loan. the `enrolledCustomer` and `newCustomer` properties are required to support the two different sources in the calling Arazzo description
+              type: object
+              required:
+                - products
+                - totalAmount
+              properties:
+                enrolledCustomer:
+                  description: The customer resource URI for a previously enrolled customer
+                  allOf:
+                    - $ref: '#/components/schemas/CustomerUri'
+                newCustomer:
+                  description: A newly-created customer resource URI for this loan
+                  allOf:
+                    - $ref: '#/components/schemas/CustomerUri'
+                products:
+                  description: Product codes for products included in loan. Supplied to ensure any special terms are included in loan agreement
+                  type: array
+                  minItems: 1
+                  items:
+                    $ref: '#/components/schemas/ProductCode'
+                totalAmount:
+                  description: Loan amount being requested
+                  allOf:
+                    - $ref: '#/components/schemas/CurrencyAndAmount'                
+      responses:
+        "202":
+          description: New loan initiated. This may require authorization before it is finalized
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - loanTransactionId
+                  - links
+                properties:
+                  redirectAuthToken:
+                    description: A token that allows the loan to be completed without further authorisation. Is omitted if authentication and authorisation by the End User is required
+                  loanTransactionId:
+                    $ref: '#/components/schemas/LoanTransactionId'
+                  links:
+                    type: object
+                    required:
+                      - self
+                    properties:                      
+                      self:
+                        description: Link to this resource
+                        type: string
```

Comment:
should this use the `customerUri` reference you have used in other response definitions?